src/mqueryfront/src/components/QueryProgressBar.js

```diff
@@ -16,74 +43,81 @@ const QueryProgressBar = (props) => {
         files_matched,
         total_datasets,
         datasets_left,
+        agents_left,
     } = job;
 
+    const getPercentage = (files) =>
+        total_files ? Math.round((files * datasetFrac * 100) / total_files) : 0;
+
     const datasetsDone = total_datasets - datasets_left;
     const datasetFrac = total_datasets > 0 ? datasetsDone / total_datasets : 0;
     const datasetPct = Math.round(datasetFrac * 100);
 
-    const getPercentage = (files) =>
-        total_files ? Math.round((files * datasetFrac * 100) / total_files) : 0;
+    // TODO: remove the "failed" status after merging #317
+    console.log(status);
```

Comment:
Leftover log, probably safe to remove?

Reply:
Oof. Looks like CI rule that was meant to catch this stopped working at some point. Added issue for myself: #337